### DISCUSSION TITLE
How to show custom q-table v-slot:bottom when table is empty?

### DISCUSSION THREAD

**Discussion by rubymongoose** [quasarframework/quasar#17408] [General - Components / Directives / etc]
Hello, I'm use v-slot:bottom in q-table for some user (inputs, buttons, etc) controls.
During the test process I noticed that v-slot:bottom not showing when table is empty.
How can I change this and show v-slot:bottom constantly?

**Comment by jaybo** [comment]
Use both the `v-slot:bottom `and `v-slot:no-data` and duplicate the content in both slots.
